api/accounts/accounts-model.js

```diff
@@ -1,21 +1,32 @@
-const getAll = () => {
-  // DO YOUR MAGIC
+const db = require(`../../data/db-config`)
+
+const getAll = async () => {
+  const allAccounts = await db('accounts')
+  return allAccounts
 }
 
-const getById = id => {
-  // DO YOUR MAGIC
+const getById = async id => {
+  const account = await db('accounts').where('id', id).first()
+  return account
 }
 
-const create = account => {
-  // DO YOUR MAGIC
+const create = async account => {
+  const newAcct = await db('accounts').insert(account)
+  const createdAcct = await getById(newAcct)
```

Comment:
Nit: You could just return await getById(newAcct) instead of creating the variable